### PR TITLE
fix(modal): add size to modal with reference to theme.sizes

### PR DESCRIPTION
**Pull request by Ryuta1346**
no issues.

## 📝 Description

- add size to modal with reference to theme.sizes.

## ⛳️ Current behavior (updates)

modal is able to set size until ‘6xl’. 

## 🚀 New behavior

> Please describe the behavior or changes this PR adds

in theme.sizes, there are the size until ‘8xl’.

## 💣 Is this a breaking change (Yes/No):

No.

### CODE REVIEW

File: .changeset/serious-panthers-brake.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@chakra-ui/modal": major
+"@chakra-ui/theme": major
```
Comment (segunadebayo) [inline .changeset/serious-panthers-brake.md:3]:
```suggestion
"@chakra-ui/modal": minor
"@chakra-ui/theme": minor
```

**Comment by segunadebayo** [comment]
Does this PR solve any inherent problem that you have?

**Comment by Ryuta1346** [comment]
@segunadebayo 
I think so.

**Comment by segunadebayo** [comment]
I'm not convinced we should add more options just because we can.

I'd much rather wait till it's a pressing need before adding it. I'll revive this PR when the time is right.

I appreciate the work here, though.

**Pull request closed without merging**